config/config.go

```diff
@@ -161,6 +163,8 @@ type Config struct {
 	EnableGlobalIndex bool `toml:"enable-global-index" json:"enable-global-index"`
 	// DeprecateIntegerDisplayWidth indicates whether deprecating the max display length for integer.
 	DeprecateIntegerDisplayWidth bool `toml:"deprecate-integer-display-length" json:"deprecate-integer-display-length"`
+	// DcLocation indicates the data center zone where the tidb server located in.
+	DcLocation string `toml:"dc-location" json:"dc-location"`
```

Comment:
Can we reuse the configuration of `Labels`? as long as the PD matches it we know think the DcLocaltion.

Reply:
we can remove this config item.